uiowa/uiowa
# LB editor theming updates and header lock.

Resolves #2029, resolves #1523, resolves #1908, resolves #2046, resolves https://github.com/uiowa/uids/issues/274

Work toward providing a closer match between the LB 'preview' available in the editor and the final produced page.

### Testing
- `composer install`
- Sync a site.
- Check that the issues described in #2029 are resolved.
- Add a local webmaster and masquerade as them. Check that the header section is locked while masquerading but not as an admin.
